### PR TITLE
Add side menu

### PR DESCRIPTION
**Pull request by SkaeX**
- Add static side menu.
- Add updated TestSeeder to have a better look.

To do:
- [x] Make side menu dynamic

### CODE REVIEW

**Comment by BrainMaestro** [comment]
Notes:
- the menu should be a top level element. Not part of the sections page. 
- your indentation level is wrong. Still 2 spaces instead of 4.

Also, we hadn't decided on the type of menu yet. Should've opened an issue where we'll decide before doing a pr

**Comment by SkaeX** [comment]
My bad @BrainMaestro, this'll wait till we decide then.

**Comment by BrainMaestro** [comment]
It's fine. Minor oversight.

**Comment by BrainMaestro** [comment]
Spacing is still wrong. Please switch it to 4 spaces. We cannot have style mismatches in the same codebase

File: resources/assets/js/app.vue [hunk 1]
```diff
@@ -32,6 +30,6 @@
 
     main {
         flex: 1 0 auto;
-        padding: 3em 0;
+        padding: 3%;
```
Comment (BrainMaestro) [inline resources/assets/js/app.vue]: Is there a reason why this was changed? I reverted

**Comment by BrainMaestro** [comment]
Look at my changes before I merge.

I know we've already decided, but for the record I do not like this side menu at all. It makes the content non-centered which looks really bad in my opinion. Just look at the sections table. Oh well.

**Comment by BrainMaestro** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/BrainMaestro/FCE/pull-request/156/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=BrainMaestro/FCE)

**Pull request merged**